app/views/repositories/_sidebar.html.erb

```diff
@@ -8,19 +8,17 @@
           </span>
         </li>
         <% repositories.each do |repository| %>
-          <% cache repository do %>
             <li class="<%= 'active parent_li' if current_page?(repository_path(repository)) %>" >
```

Comment:
You should decrease indent on whole block for 2 whitespaces... Otherwise, this is ready for merge.